common/src/cams/orders.test.ts

```diff
@@ -26,11 +26,16 @@ describe('orders model tests', () => {
   });
 
   describe('temporary mapper function tests', () => {
+    const restMetaDataFields = ['_actions'];
```

Comment:
This feels like it is in the wrong place. The rest action should be bound late and should not be part of the domain model.